index.js

```diff
@@ -1,61 +1,68 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable class-methods-use-this */
+/* eslint-disable no-use-before-define */
+
 class Book {
```

Comment:
[OPTIONAL]
- [ ] Instead of disabling these eslint rules, try to fix the errors and in the process, you'll learn new concepts.
- [ ] Always use the present tense for your commit messages and make your commit messages more meaningful. One of your commit messages was written in the past tense.